File: Questions/Java/Question5.java
// Question 5
// Check if the sentence is Pangram.
// A Pangram is a sentence where every letter of the English alphabet appears at least once.

public class Question5 {
    public boolean checkIfPangram(String sentence) {
        // Write
        // Your
        // Code
        // Here
    }

    public static void main(String[] args) {
        Question5 q5 = new Question5();

        // Test Case 1: Pangram
        String sentence1 = "thequickbrownfoxjumpsoverthelazydog";
        System.out.println("Test Case 1: " + q5.checkIfPangram(sentence1)); // Expected output: true

        // Test Case 2: Not a pangram
        String sentence2 = "thequickbrownfoxjumpsoverthelazyg";
        System.out.println("Test Case 2: " + q5.checkIfPangram(sentence2)); // Expected output: false

        // Test Case 3: Empty string
        String sentence3 = "";
        System.out.println("Test Case 3: " + q5.checkIfPangram(sentence3)); // Expected output: false

        // Test Case 4: String with only one letter
        String sentence4 = "a1";
        System.out.println("Test Case 4: " + q5.checkIfPangram(sentence4)); // Expected output: false

        // Test Case 5: String with only one letter
        String sentence5 = "abcdefghijklmnopqrstuvwxyz";
        System.out.println("Test Case 5: " + q5.checkIfPangram(sentence5)); // Expected output: true

    }
}

File: Questions/C/Question5.c
#include <stdio.h>
#include <stdbool.h>
#include <ctype.h>

// Check if the sentence is Pangram.
// A Pangram is a sentence where every letter of the English alphabet appears at least once.

bool checkIfPangram(const char *sentence)
{
    // Write Your Code Here
}

int main()
{
    // Test Case 1
    const char *sentence1 = "thequickbrownfoxjumpsoverthelazydog";
    printf("Test Case 1: %s\n", checkIfPangram(sentence1) ? "true" : "false");

    // Test Case 2
    const char *sentence2 = "thequickbrownfoxjumpsoverthelazyg";
    printf("Test Case 2: %s\n", checkIfPangram(sentence2) ? "true" : "false");

    // Test Case 3
    const char *sentence3 = "";
    printf("Test Case 3: %s\n", checkIfPangram(sentence3) ? "true" : "false");

    // Test Case 4
    const char *sentence4 = "a1";
    printf("Test Case 4: %s\n", checkIfPangram(sentence4) ? "true" : "false");

    // Test Case 5
    const char *sentence5 = "abcdefghijklmnopqrstuvwxyz";
    printf("Test Case 5: %s\n", checkIfPangram(sentence5) ? "true" : "false");

    return 0;
}
